types/pixi.js/index.d.ts

```diff
@@ -3006,14 +4153,23 @@ declare namespace PIXI {
              * Alias method for `removeListener`
              */
             //tslint:disable-next-line:ban-types forbidden-types
-            off(event: string | symbol, fn?: Function, context?: any, once?: boolean): this;
+            off(
+                event: string | symbol,
+                fn?: Function,
+                context?: any,
+                once?: boolean
+            ): this;
             /**
              * Alias method for `on`
              */
             //tslint:disable-next-line:ban-types forbidden-types
-            addListener(event: string | symbol, fn: Function, context?: any): this;
+            addListener(
+                event: string | symbol,
+                fn: Function,
+                context?: any
+            ): this;
             /**
-             * This function doesn't apply anymore.
+             * This function doesn"t apply anymore.
```

Comment:
looks like find/replace ran out of control here

Reply:
It did! Thank you for the heads up! , it is fixed for a future update but for now is fine.